projects/log_listener/src/log_listener/listener.py

```diff
@@ -49,12 +58,24 @@ def tercc(self) -> EiffelTestExecutionRecipeCollectionCreatedEvent:
             self.__tercc = tercc
         return self.__tercc
 
-    def new_message(self, message: dict, _: Optional[str] = None) -> None:
-        """Handle new log messages from ETOS."""
-        self.logger.info(message)
+    def new_event(self, event: dict, _: Optional[str] = None) -> None:
+        """Get a new event from the internal RabbitMQ bus and write it to file."""
+        if event.get("event") is None:
+            event = {"event": "message", "data": event}
+        self.__write(**event)
+
+    def __write(self, event: str, data: str) -> None:
+        """Write an event, and its data, to a file."""
         with self.lock:
-            with self.log_file.open("a") as log_file:
-                log_file.write(f"{json.dumps(message)}\n")
+            data = {"id": self.id, "event": event, "data": data}
+            with self.event_file.open("a") as events:
+                events.write(f"{json.dumps(data)}\n")
+            # TODO: Temporarily writing to two files, this next one shall be removed when the
```

Comment:
NIT: Why not refer to the `log_file` using it's name?
```suggestion
            # TODO: Temporarily writing to two files, the log_file shall be removed when the
```